oracles/src/domain/knowledge_base/utils.py

```diff
@@ -0,0 +1,11 @@
+import json
+from typing import List
+from src.domain.knowledge_base.entities import Document
+
+
+def deserialize_documents(documents_str: str) -> List[Document]:
```

Comment:
Sometimes I really wish python had typing for raising exceptions :D

Reply:
Java-like `throws`?